presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -705,3 +706,23 @@ void PeriodicTaskManager::detachWorker() {
 }
 
 } // namespace facebook::presto
+
+template <>
+struct fmt::formatter<facebook::velox::memory::MemoryArbitrator::Stats>
```

Comment:
Why do we need this? Shouldn't this type of change go into wherever MemoryArbitrator::Stats are defined? If Velox is later updated to add a similar change, will this cause a conflict?

Reply:
Good point! @kgpai do you know the answer to this?

Reply:
I think these should live in Velox. Added a PR here https://github.com/facebookincubator/velox/pull/8608